app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,52 @@
+class OrganizationsController < ApplicationController
+  def index
+    @organizations = Organization.order('name ASC')
+  end
+
+  def show
+    @organization = Organization.find(params[:id])
+  end
+
+  def new
+    @organization = Organization.new
+  end
+
+  def edit
+    @organization = Organization.find(params[:id])
+  end
+
+  def create
+    set_user
+
+    @organization = @user.organizations.create(organization_params)
+
+    if @organization.save
+      redirect_to current_user
+    else
+      redirect_to "/users/#{current_user.id}/organizations/new"
+    end
+  end
+
+  def update
+    if @organization.update(organization_params)
+      redirect_to organization_path
+    else
+      redirect_to edit_organization_path
+    end
+  end
+
+  def destroy
```

Comment:
Extra space after def
```suggestion
  def destroy
```